lib/ui/view/channel/channel_info/widgets/third_section.dart

```diff
@@ -36,9 +37,17 @@ class ThirdSection extends StatelessWidget {
                   left: 11,
                   top: 19,
                 ),
-                child: Text(
-                  'Members (5)',
-                  style: descriptionStyle(),
+                child: GestureDetector(
+                  onTap: () {
+                    Navigator.push(
```

Comment:
There should be no navigation or logic in your UI code, make this in the view model, use the navigation services provided,

Reply:
Changes have been made and commits have been pushed. Kindly take a look